src/status_im/notifications/local.cljs

```diff
@@ -189,10 +190,13 @@
            local-push-android))
 
 (fx/defn process
-  [cofx evt]
-  (if platform/ios?
-    {::local-push-ios evt}
-    (local-notification-android cofx evt)))
+  [cofx evt-js]
+  (when (and (= (get-in cofx [:db :app-state]) "inactive")
+             (get-in cofx [:db :multiaccount :remote-push-notifications-enabled?]))
```

Comment:
Not sure this is correct, on android this will work and `remove-push-notifications-enabled?` is only for IOS.

What's the reason for these changes? 
If it's just performance https://github.com/status-im/status-react/issues/11806 should help, as we check on status-go if notifications are enabled and it moves all the processing for messages in status-go (not for transactions yet).

Reply:
ok, i'll remove this condition then, thanks